services/keywords/main.py

```diff
@@ -11,13 +12,21 @@ def get_keywords_service(request):
         json of language detected and keywords for text
     """
     json = request.get_json()
+    logging.info("Endpoint request:", json)  # pylint: disable=logging-too-many-args
+
     if json is None or "text" not in json:
-        return {"error": {"message": "ValueError: Expected 'text' field in json body is missing"}}
+        error = {"error": {"message": "ValueError: Expected 'text' field in json body is missing"}}
+        logging.error(error['error']['message'])  # pylint: disable=logging-too-many-args
```

Comment:
This would be more readable if you made `message` a variable and used it in both places.